scriptworker/config.py

```diff
@@ -65,24 +65,27 @@ def unfreeze_values(dictionary):
     A recursive function(top-down conversion)
```

Comment:
I noticed some small bugs in `freeze_values()`:

* Like you did at [line 73](https://github.com/mozilla-releng/scriptworker/pull/61/files#diff-93b3cf2396766ce44e10261ffca70d55R73), we need to check for instances of tuples and frozendicts as well. Otherwise if a tuple contains a list, that list won't be frozen.
* line 55: `freeze_values(idx)` has to become `freeze_values(dictionary[idx])`, right?
* the top level structure is not frozen.

This makes wonder if we shouldn't make `freeze_values()` pure. The code would be a couple of line long, and the top-level gets frozen as well. The only disadvantage I see: each time we call freeze_values() somewhere, the next line has to be `del my_var_which_just_got_copied_by_freeze_values`